crates/subspace-networking/src/node.rs

```diff
@@ -212,14 +212,16 @@ impl Node {
     /// running in the relay server mode (which happens automatically when addresses to listen on
     /// are provided).
```

Comment:
This comment is out of date now

Reply:
Fixed and added a new error type on non-relay-server spawning call.